roles/tas_single_node/README.md

```diff
@@ -268,6 +269,12 @@ Deploy the [RHTAS](https://docs.redhat.com/en/documentation/red_hat_trusted_arti
 | username | Username for the cockpit user. | str | no |  |
 | password | Password for the cockpit user. | str | yes |  |
 
+#### Options for main > tas_single_node_trust_root
+
+|Option|Description|Type|Required|Default|
+|---|---|---|---|---|
+| full_archive | A compressed base64-encoded .tgz file of the trust root. This archive file has a single directory named `repository/`, containing the contents of the TUF repository, for example `1.root.json`. The default value is an empty string. Using the default value generates the trust root content, unless a trust root already exists. Specifying a `full_archive` string removes any earlier configured trust root content, and starts to use the new specified content. If changing this value back to an empty string after setting the `full_archive` option, then we continue serving the previous value for the trust root content. To reset the trust root content, you must remove all files in the volume associated with the `tuf-repository` pod. | str | no |  |
```

Comment:
@[bkabrda@redhat.com](mailto:bkabrda@redhat.com) this is different from what we have in operator. We do not have an option to provide the full_archive TUF repository. The TUF configuration consumes keys with metadata instead. The key rotation is done on the live container. Can we take a similar approach to unify both the configuration?

Reply:
@bouskaJ we discussed this with others and came to the conclusion that this is a more Ansible-y approach and is easier on users; we think that it is reasonable to diverge in this case. Going forward, the TUF auto updater feature is going to be configured similarly on both the operator and the collection (it will be mutually exclusive with this setting).

We think this is a more practical alternative to copying the data over manually (do note that this is something that the user is supposed to use to provide manually updated trust root; the initial trust root still gets generated automatically - same as for operator)

Reply:
Ok, it seems that this is not the solution taken in the first second. :-D I understand that we will need to differ in some cases. I just wanted to point that out because once we create different APIs, it will be hard to unify them.

Reply:
I totally get your point and I do agree with you that we need to strive for consistency; however, we might see cases where it's beneficial to slightly diverge and I do believe this is one of them. I always welcome more opinions, so thanks a lot either way!